src/pages/Preferences.vue

```diff
@@ -11,6 +11,7 @@ import PreferencesBody from '@/components/Preferences/ModalBody.vue';
 import PreferencesHeader from '@/components/Preferences/ModalHeader.vue';
 import PreferencesNav from '@/components/Preferences/ModalNav.vue';
 import type { ServerState } from '@/main/commandServer/httpCommandServer';
+import { Settings, load } from '~/config/settings';
```

Comment:
We tend to use `@` over `~` for some mysterious reason. (I actually don't know why.)

Reply:
Thanks @mook-as , I fixed it.